### PR TITLE
Update ExceptionExercises

### PR DESCRIPTION
**Pull request by chanjk**
- Update comments
- Update test descriptions
- Allow Lists and our custom Exceptions to be compared structurally in
`collectErrors` test

P.S. I don't know if the test for `collectErrors` is meant to fail by design. If so, please let me know/feel free to remove the implicits and the `Equivalence` import.

### CODE REVIEW

**Comment by ssanj** [comment]
I might have to put a pause on some of these because I fixed them in PR #42.

File: src/main/scala/fundamentals/level03/ExceptionExercises.scala [hunk 1]
```diff
@@ -77,16 +76,16 @@ object ExceptionExercises {
     * What does this imply?
     *
     * scala> createPerson("Fred", "32")
-    * = "Person(Fred, 32)"
+    * = Person("Fred", 32)
```
Comment (ssanj) [inline src/main/scala/fundamentals/level03/ExceptionExercises.scala]: This is supposed to be output from the REPL. Strings are not quoted.

File: src/test/scala/fundamentals/level03/ExceptionExercisesTest.scala [hunk 2]
```diff
@@ -93,21 +93,37 @@ class ExceptionExercisesTest extends FunSpec with TypeCheckedTripleEquals {
...
+    it("should return a List of Person instances") {
       assert(createValidPeople2 === List(Person("Tokyo", 30), Person("Berlin", 43)))
     }
   }
 
   describe("collectErrors") {
 
-    it("should return a List Exceptions thrown while processing inputs") {
+    // structural equality for our custom Exceptions
+    implicit val exceptionEq: Equivalence[Exception] = new Equivalence[Exception] {
```
Comment (ssanj) [inline src/test/scala/fundamentals/level03/ExceptionExercisesTest.scala]: I implemented a slightly simpler equality check in #42.

**Comment by ssanj** [comment]
I've rolled your typo fixes into #42 and fixed the `collectErrors` test in a simpler way. Thanks for your hard work! 👍

**Comment by chanjk** [comment]
No worries, and thanks for looking into the PR! 👍

**Pull request closed without merging**